test/runnable/importc-test1.sh

```diff
@@ -8,8 +8,19 @@ else
 	cp ${EXTRA_FILES}/importc_test.i.in ${EXTRA_FILES}/importc_test.i
 fi
 
-$DMD -m${MODEL} -I${OUTPUT_BASE} -of${OUTPUT_BASE}${EXE} ${EXTRA_FILES}${SEP}importc_main.d ${EXTRA_FILES}/importc_test.i
+# Case1: The referenced .i is passed on commandline
+$DMD -m${MODEL} -I${OUTPUT_BASE} -of${OUTPUT_BASE}${EXE} ${EXTRA_FILES}${SEP}importc_main.d ${EXTRA_FILES}${SEP}importc_test.i
 
 ${OUTPUT_BASE}${EXE}
 
-rm_retry ${OUTPUT_BASE}{a${OBJ},${EXE}}
+# Case2: The referenced module compiled from an .i file is NOT passed on commandline
+#        and the compiler has to guess the right name.
+#        Note: object is passed to keep linker happy
+#        Note: if the module object created from the .i file just contains DECLARATIONS, you can omit the .o file.
+#              There is no code to link in this case. This work e.g. for preprocessed C headers, e.g. /usr/include/zstd.h
+$DMD -c -m${MODEL} -I${OUTPUT_BASE} ${EXTRA_FILES}${SEP}importc_test.i -of=${EXTRA_FILES}${SEP}importc_test.o
+$DMD -m${MODEL} -I${OUTPUT_BASE} -of${OUTPUT_BASE}${EXE} -I${EXTRA_FILES} ${EXTRA_FILES}${SEP}importc_test.o ${EXTRA_FILES}${SEP}importc_main2.d
+
+${OUTPUT_BASE}${EXE}
+
+rm_retry ${OUTPUT_BASE}{a${OBJ},${EXE}} ${EXTRA_FILES}${SEP}importc_test.o ${EXTRA_FILES}/importc_test.i
```

Comment:
This fail on windows, I suspect because you have used `.o` here instead of `.obj`. I think you want `.${OBJ}`l but I'm not certain.

Reply:
Yes, I will change that to ${OBJ} and test it again.